gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/02_add_shared_policy_group.yml

```diff
@@ -28,11 +28,21 @@ databaseChangeLog:
             tableName: ${gravitee_prefix}sharedpolicygroups
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.